src/styles/_catalog.scss

```diff
@@ -11,9 +11,16 @@
     align-items: center;
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $card-border-color;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform 0.3s ease, color 0.3s ease;
```

Comment:
you can create a variable for the same transition-duration here

Reply:
Thx, will change this soon